index.d.ts

```diff
@@ -1766,15 +1766,23 @@ declare namespace Shopify {
     processed_at: string;
   }
 
+  interface IInventoryItemCountryHarmonizedSystemCode {
+    harmonized_system_code: string | null;
+    country_code: string | null;
```

Comment:
```suggestion
    harmonized_system_code: string;
    country_code: string;
```

I think these can't be `null`